File: Backend/LinkCare/src/main/java/com/ssafy/linkcare/group/dto/InvitationResponse.java
package com.ssafy.linkcare.group.dto;

import java.time.LocalDateTime;

/*
    * 초대 링크 응답 DTO
*/
public record InvitationResponse(
        Long invitationSeq,
        Long groupSeq,
        String groupName,
        String invitationToken,
        LocalDateTime createdAt,
        LocalDateTime expiredAt,
        String invitationUrl  // 프론트엔드에서 사용할 전체 URL
) {}

File: Frontend/app/src/main/java/com/a307/linkcare/feature/commongroup/data/model/response/InvitationResponse.kt
package com.a307.linkcare.feature.commongroup.data.model.response

import com.google.gson.annotations.SerializedName

data class InvitationResponse(
    @SerializedName("invitationSeq")
    val invitationSeq: Long,

    @SerializedName("groupSeq")
    val groupSeq: Long,

    @SerializedName("groupName")
    val groupName: String,

    @SerializedName("invitationToken")
    val invitationToken: String,

    @SerializedName("createdAt")
    val createdAt: String,

    @SerializedName("expiredAt")
    val expiredAt: String,

    @SerializedName("invitationUrl")
    val invitationUrl: String?
)
